visual-studio-dev-vm-O365/azuredeploy.json

```diff
@@ -0,0 +1,275 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.2",
+  "parameters": {
+    "storageType": {
```

Comment:
Move this to a variable along with the vmSize - since they go hand in hand in the premium case and the user won't know how to match them.